### DISCUSSION TITLE
add info about me

### DISCUSSION THREAD

**Discussion by Nerverjoy** [community/community#20280] [Codespaces]
Add on optional extended description...

**Comment by diegoleondev** [comment]
Hi @Nerverjoy, This article explains step by step how to create the section: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme

**Comment by surajsahani** [comment]
Good question.
Follow this steps.
1. create a repo name should be same as your username, example username = JohnDoe, repo name = JohnDoe.
2.  then create a readme.md file inside it than. it will automatically display at your profile page.

One optional 
if you have more content to show you can simply use 

```
<details>
  <summary markdown="span">something </summary>
</details>
```
tag and hide all trivial details inside it. I would love to see how you personalized your readme
